physionet-django/project/modelcomponents/publishedproject.py

```diff
@@ -206,16 +206,22 @@ def file_display_url(self, subdir, file):
         return reverse('display_published_project_file',
             args=(self.slug, self.version, os.path.join(subdir, file)))
 
-    def has_access(self, user):
+    def has_access_to_project_and_files(self, user):
         """
         Whether the user has access to this project
         The logic should mirror PublishedProjectManager#accessible_by,
         but for a specific project.
         """
```

Comment:
Now this comment is duplicated (it's in both methods). It should only be in the new one (the one that does not check `allow_file_downloads`